src/main/host/descriptor/file.h

```diff
@@ -50,10 +53,13 @@ int file_openat(File* file, File* dir, const char* pathname, int flags, mode_t m
 // ************************
 
 /* Returns the flags that were used to open the file. */
-int file_getFlags(File* file);
+int file_getOriginalFlags(File* file);
 
 /* Returns the mode that was used to open the file. */
-mode_t file_getMode(File* file);
+mode_t file_getOriginalMode(File* file);
+
+/* Get only the flags that shadow emulates. Will be a subset of EMULATED_FLAG_MASK. */
```

Comment:
This comment is also a little unclear, though maybe expanding the earlier comment to describe what you mean by 'emulate' will help address it.

Reply:
Expanded the comment.